dropshot_endpoint/src/lib.rs

```diff
@@ -212,7 +212,7 @@ fn do_endpoint(
         // ... a constant of type `#name` whose identifier is also #name
         #[allow(non_upper_case_globals, missing_docs)]
         #description_doc_comment
-        const #name: #name = #name {};
+        pub const #name: #name = #name {};
```

Comment:
How about we look at the function and mirror its visibility? This should be accessible through `ast.vis`

Reply:
Sorry for the delay. I've implemented the mirror in visibility, cleaned up the resulting test errors, and updated the related example.